botnim/sync.py

```diff
@@ -52,115 +52,62 @@ def openapi_to_tools(openapi_spec):
     return ret
 
 def update_assistant(config, config_dir, production, replace_context=False):
-    tool_resources = None
-    tools = None
-    print(f'Updating assistant: {config["name"]}')
-    # Load context, if necessary
-    if config.get('context'):
-        for context_ in config['context']:
-            name = context_['name']
-            if not production:
-                name += ' - פיתוח'
-            vector_store = client.beta.vector_stores.list()
```

Comment:
This part also disappeared - what if I don't want to update the context?

Reply:
to be fixed, with a better implementation.